ansible/roles/cni/tasks/cilium.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Install Helm v3
+  shell: |
+    curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+  args:
+    warn: false
+
+- name: Add Cilium Repo
+  command:
+    cmd: helm repo add cilium https://helm.cilium.io/
+
+- name: Deploy Cilium
+  shell: |
+    helm upgrade -i cilium cilium/cilium --version {{ cilium_helm_version }} \
+    --set global.registry="docker.io/cilium" \
+    --set global.tag="{{ cilium_image_version }}" \
+    --set global.tunnel="disabled" \
+    --set global.externalIPs.enabled="true" \
+    --set global.ipam.operator.clusterPoolIPv4PodCIDR="{{ cluster_pod_subnet }}" \
+    --set global.ipam.operator.clusterPoolIPv4MaskSize="24" \
+    --set global.endpointRoutes.enabled="true" \
+    --set global.hostServices.enabled="true" \
+    --set global.autoDirectNodeRoutes="true" \
+    --set global.nodePort.enabled="true" \
+    --set global.nodePort.mode="dsr" \
+    --set global.masquerade="false" \
+    --set global.hubble.enabled="true" \
+    --set global.hubble.ui.enabled="true" \
+    --set global.hubble.relay.enabled="true" \
```

Comment:
I wonder if there is an easier way to store these values to allow them to be easier to configure for users, for example, a user using arm might not be able to enable hubble right now. Would it make sense if we stored these in a values.yaml under files/ and we copy that and just `--values=values.yaml` on the host, this is just a thought?

Reply:
yeah we could template out a cilium-values.yaml file.